matcher.py

```diff
@@ -32,7 +37,14 @@ def lexicalMatch(ref, ex, ignoreStopwords, ignoreCase):
         sRef = ref.bow().split(' ')
         sEx = ex.bow().split(' ')
         count = 0
-        
+        gold_predicate_words = [lemmatizer.lemmatize(w) for w in ref.elementToStr(ref.pred.strip()).split(' ')]
+        ex_predicate_words = [lemmatizer.lemmatize(w) for w in ex.elementToStr(ex.pred.strip()).split(' ')]
+
+        # Make sure the predicates, at the very least, match before 
+        # seeing if the doing the lexical match
+        if not bool(set(ex_predicate_words) & set(gold_predicate_words)):
```

Comment:
Ensure that, at the very least,  one of the predicate words matches one of the gold predicate words using set intersection. If not, you can immediately say these two extractions are not a good match.